.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!-- First time contributors: Take a moment to review CONTRIBUTING.md! -->
```

Comment:
In this case there's no CONTRIBUTING.md.

The equivalent for setup.py is the Developer's guide: 

https://setuptools.readthedocs.io/en/latest/developer-guide.html